spec/test.spec.js

```diff
@@ -133,7 +133,7 @@ describe('S3Adapter tests', () => {
     });
 
     it('should accept options and overrides as an Enviromental Variable', () => {
-      var confObj = { bucketPrefix: 'test/', bucket: 'bucket-1', secretKey: 'secret-1', accessKey: 'key-1' };
+      var confObj = { bucketPrefix: 'test/', bucket: 'bucket-1', secretKey: 'secret-1', accessKey: 'key-1'};
```

Comment:
why'd you take out that trailing space?  for consistency, there should be a space between the `1' }`